interface/src/Application.cpp

```diff
@@ -5545,8 +5545,8 @@ bool Application::importEntities(const QString& urlOrFilename, const bool isObse
     return success;
 }
 
-QVector<EntityItemID> Application::pasteEntities(float x, float y, float z) {
-    return _entityClipboard->sendEntities(&_entityEditSender, getEntities()->getTree(), x, y, z);
+QVector<EntityItemID> Application::pasteEntities(const QString entityHostType, const float x, const float y, const float z) {
```

Comment:
const QString&, not just const QString

and actually it’s better to leave floats as just floats!  primitives and small objects are better left alone, the copy is actually less expensive than the conversion to a reference (which is actually a pointer).